src/parser.js

```diff
@@ -685,7 +693,10 @@ export default class Parser {
         let unbalanced = 1;
         this.position ++;
         if (last && last.type === types.PSEUDO) {
-            const selector = new Selector({source: {start: tokenStart(this.tokens[this.position - 1])}});
```

Comment:
As far as I can tell this was always incorrect.

`this.tokens[this.position - 1]` is the open parenthesis, but that isn't the start token of the selector. The selector starts inside the parenthesis.

```
:is(foo)
    ^ here

:is(foo)
   ^ not here
```

By comparison when the selector is part of a list and follows a comma it also does not have the comma as the start position. It starts after the comma.

```
:is(foo,bar)
        ^ here

:is(foo,bar)
       ^ not here
```